python/triton_kernels/bench/bench_mlp.py

```diff
@@ -130,24 +130,31 @@ def roofline_mlp(batch_sizes, dim1, dim2, n_expts_tot, n_expts_act, x_dtype, w_d
         # torchrun --nproc-per-node=4 ./bench_mlp.py --tp 1 --ep 4 --name gpt-oss-x2 --quantized
         # torchrun --nproc-per-node=4 ./bench_mlp.py --tp 4 --ep 1 --name gpt-oss-x2 --quantized
         # torchrun --nproc-per-node=4 ./bench_mlp.py --tp 4 --ep 1 --name dense --quantized
-        argparse = argparse.ArgumentParser()
-        argparse.add_argument("--tp", type=int, default=1)
-        argparse.add_argument("--ep", type=int, default=1)
-        argparse.add_argument("--name", type=str, choices=["dense", "gpt-oss-x2"])
-        argparse.add_argument("--quantized", action="store_true", default=False)
-        args = argparse.parse_args()
-        dtypes = dense_dtypes if args.quantized else quantized_dtypes
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--tp", type=int, default=1)
+        parser.add_argument("--ep", type=int, default=1)
+        parser.add_argument("--name", type=str, choices=["dense", "gpt-oss-x2"])
+        parser.add_argument("--quantized", action="store_true", default=False)
+        args = parser.parse_args()
+        dtypes = quantized_dtypes if args.quantized else dense_dtypes
         if args.name == "dense":
             assert args.ep == 1, "EP must be 1 for dense"
-            roofline_mlp(batch_sizes_dense, 8192, 8192, 1, 1, *dtypes, TP=args.tp, EP=args.ep, name="dense")
+            roofline_mlp(batch_sizes_dense, 8192, 8192, 1, 1, dtypes[0], dtypes[1], TP=args.tp, EP=args.ep,
```

Comment:
It's better to explicitly unpack the dtype array, otherwise mypy will raise a complain as we unpack before a keyword argument